src/shared/interfaces.ts

```diff
@@ -2,3 +2,9 @@ export enum AuthPageName {
     LOGIN = '/login',
     SIGNUP = '/signup',
 }
+
+export interface IServerConfiguration {
+    baseURL: string;
+    timeout: number;
+    headers: object;
+}
```

Comment:
Please add `interfaces` folder and in this folder create files with specific entity (entity groups) interfaces